mitosheet/css/endo/CellEditor.css

```diff
@@ -41,6 +41,12 @@
     position: absolute;
 }
 
+.jp-RenderedHTMLCommon .mito-container a.cell-editor-dropdown-link {
+    margin: 0.5em;
```

Comment:
Across the codebase right now, we only use `px` instead of `em`. It probably isn't best practice (in the trymito.io codebase we use `em` instead), but for consistency right now, I think it might be best to stick with `px` just because other things are hardcoded for the correct `px` settings currently. 

At some point we should do a css refactor, it needs a bit of care.